metpy/plots/tests/test_station_plot.py

```diff
@@ -341,3 +342,21 @@ def test_barb_unit_conversion_exception(u, v):
     stnplot = StationPlot(ax, x_pos, y_pos)
     with pytest.raises(ValueError):
         stnplot.plot_barb(u, v, plot_units='knots')
+
+
+@pytest.mark.mpl_image_compare(tolerance={'1.5': 0.0, '1.4': 0.0}.get(MPL_VERSION, 0.000),
```

Comment:
Given that we only support matplotlib >= 2.0, you should be able to just make this tolerance=0.